packages/odyssey-react/src/components/Infobox/Infobox.theme.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Root
+  FontLineLengthMax: theme.FontLineLengthMax,
+  SpacePadding: theme.SpaceRemM,
+  SpaceMarginBlockEnd: theme.SpaceRemM,
+  BorderRadius: theme.BorderRadiusBase,
+  ColumnGap: theme.SpaceRemS,
+
+  IconSize: theme.FontSizeHeading5,
+  IconLineHeight: theme.FontLineHeightTitle,
+
+  ContentSpaceMarginBlockEnd: theme.SpaceRemXs,
+
+  // Variants
+  CautionBackgroundColor: theme.ColorBackgroundCautionLight,
+  DangerBackgroundColor: theme.ColorBackgroundDangerLight,
+  InfoBackgroundColor: theme.ColorBackgroundPrimaryLight,
+  SuccessBackgroundColor: theme.ColorBackgroundSuccessLight,
```

Comment:
names are exhausting. This structure doesn't seem to jive with the original writeup from @arnoldsandoval-okta or what I have in Button but lets not worry for now.

https://docs.google.com/document/d/1ZXQahqi_ihNnHIGqE_VoruOzt5CVWmgnJq2_niHI51w/edit

Reply:
Easy change, I had my word order backward